test/parallel/test-assert-deep.js

```diff
@@ -127,7 +134,7 @@ test('date', () => {
       code: 'ERR_ASSERTION',
       message: `${defaultMsgStartFull}\n\n` +
               '+ 2016-01-01T00:00:00.000Z\n- MyDate 2016-01-01T00:00:00.000Z' +
-              " {\n-   '0': '1'\n- }"
+              " {\n-   '0': '1'\n- }\n"
```

Comment:
nit: I've noticed that we now have several changes related to the trailing newline, was this introduced intentionally? 😁

Reply:
@pmarchini yessir, it is point 5 and it got approved by @BridgeAR as well :)